pkgs/applications/misc/oil/default.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenv, lib, fetchurl, jq, peco, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  pname = "oil-buku";
+  version = "0.3.2";
+
+  src = fetchurl {
+    url = "https://github.com/AndreiUlmeyda/oil/archive/refs/tags/${version}.tar.gz";
+    sha256 = "1nwas1v0z2sjfab8ada9k9pf4jw4jn79rzmmn4n69xlfzzf4npq6";
+  };
+
+  passthru.updateScript = [ ./update.sh ];
+
+  postUnpack = ''
+    sed -i 's/$LIBDIR/$LOCALLIB/g' oil-${version}/src/oil
+  '';
+
+  buildInputs = [ jq peco ];
+  nativeBuildInputs = [ makeWrapper ];
+  dontPatchShebangs = true;
+
+  buildPhase = ''
+    runHook preBuild;
+    runHook postBuild;
+  '';
+
+  installPhase = ''
```

Comment:
Is there a reason we can't use the makefile from upstream? That way if something changes in the install process we don't have to manually edit this. Would also look a bit cleaner i think

Reply:
Well, I can if I set the BINDIR and LIBDIR variable, but it had thrown an error when I used the makefile even with those. I have to troubleshoot, then maybe it is possible.

Reply:
Mhmm, maybe you could `substituteInPlace` those as well?